### DISCUSSION TITLE
flomoplus v1.3.1

### DISCUSSION THREAD

**Discussion by extrastu** [it5200/flomoPlus#44] [General]
flomoPlus 新版本v1.3.1发布

- 新增保存图片到flomo(目前限制是2m)
- 解决网页批注图标问题

![image](https://user-images.githubusercontent.com/19451694/121981300-fa7ee580-cdbf-11eb-9e8d-8dbf366d0477.png)
![image](https://user-images.githubusercontent.com/19451694/121981329-04a0e400-cdc0-11eb-9a00-c11522fb8f9e.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/it5200/flomoPlugin/releases/tag/'>flomoplus v1.3.1</a>.</em>

**Comment by kokdemo** [comment]
有空了辛苦push一下新的commit哈
